### PR TITLE
add user login and profile

### PR DESCRIPTION
**Pull request by harunurhan**
* Implement user profile.

### CODE REVIEW

File: public/login_success.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+          payload: loggedInUser,
+          status: 'success'
+        },
+        window.opener.location
+      );
+    };
+  </script>
+</body>
+
+</html>
```
Comment (drjova) [inline public/login_success.html]: new line here

File: src/user/containers/LoginPage/LoginPage.scss [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  padding-top: 36px;
+  .login-button {
+    height: 52px;
+    .logo {
+      width: 36px;
+      height: 36px;
+      margin-right: 8px;
+    }
+  }
+}
```
Comment (drjova) [inline src/user/containers/LoginPage/LoginPage.scss]: better to use tachyon :P ⚛️

File: src/user/index.jsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+import { Route } from 'react-router-dom';
+
+import './index.scss';
+import LoginPage from './containers/LoginPage';
+import ProfilePage from './containers/ProfilePage';
+
+class User extends Component {
+  render() {
+    return (
+      <div className="__User__">
```
Comment (drjova) [inline src/user/index.jsx]: ```w-100```

File: src/user/loginInNewTab.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+      } else {
+        loginTab.close();
+        reject(new Error('Unauthorized'));
+      }
+    });
+  });
+}
+
+const ORCID_OAUTH_URL =
+  '//localhost:5000/oauth/login/orcid?next=login_success.html';
```
Comment (drjova) [inline src/user/loginInNewTab.js]: why is hardcoded?

File: public/login_success.html (hunk 1 above)
Comment (harunurhan) [reply public/login_success.html]: this file will be gone when it's final state

File: src/user/loginInNewTab.js (hunk 4 above)
Comment (harunurhan) [reply src/user/loginInNewTab.js]: this will be changed when it's final state.

**Pull request merged**